rfc/src/rfcs/0003-cover-statement.md

```diff
@@ -85,7 +85,16 @@ fn foo() {
 }
 ```
 
-We can consider adding an option that would cause verification to fail if a cover property was unsatisfiable or unreachable, e.g. `--fail-uncoverable`.
+The `--fail-uncoverable` option will allow users to fail the verification if a cover property is unsatisfiable or unreachable.
```

Comment:
Looks like the @adpaco-aws from the past already took that into account so the draft implementation in #2532 includes test cases for both and, when `--fail-uncoverable` is enabled, they fail as expected. That's because the current implementation checks that all `cover` properties are satisfiable for the `fail_uncoverable` condition to pass. Any other values in `cover` properties will result in failure.

Reply:
What I don't like so much about the draft implementation are the words being used. For example, there is a test in `fail-uncoverable/uncoverable/test.rs` with the description ``Checks that the `--fail-uncoverable` option causes a failure due to a cover statement not being covered.``. I don't like the terms `uncoverable` and `being covered`  so much because they're ambiguous (does `covered` mean reachable or satisfiable?). I'll make a revision later today.